src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -548,24 +564,51 @@ protected void schedule(GerritCause cause, GerritTriggeredEvent event) {
      * @param event   the event.
      * @param project the project to build.
      * @deprecated
-     *    moved to {@link EventListener#schedule(GerritTrigger, GerritCause, GerritTriggeredEvent, AbstractProject)}
+     *    moved to {@link EventListener#schedule(GerritTrigger, GerritCause, GerritTriggeredEvent, Job)}
      */
     @Deprecated
-    protected void schedule(GerritCause cause, GerritTriggeredEvent event, AbstractProject project) {
+    protected void schedule(GerritCause cause, GerritTriggeredEvent event, Job project) {
         createListener().schedule(this, cause, event, project);
     }
 
     /**
+     * Schedules a build of a job.
+     * <p>
+     * Added here to facilitate unit testing.
+     *
+     * @param theJob The job.
+     * @param quitePeriod Quite period.
+     * @param cause Build cause.
+     * @param badgeAction build badge action.
+     * @param parameters Build parameters.
+     * @return Scheduled build future.
+     */
+    protected Future schedule(final Job theJob, int quitePeriod, GerritCause cause, BadgeAction badgeAction,
```

Comment:
Right, but putting it anywhere else makes unit testing a total PITA. I was around the house on this one trying to make it fit into the listener, but doing so was a total pain. [See this comment](https://github.com/jenkinsci/gerrit-trigger-plugin/pull/240/files#diff-462a2e225475f498620d2751fa464ca0R599).